lms/djangoapps/discussion/rest_api/serializers.py

```diff
@@ -208,6 +228,7 @@ class ThreadSerializer(_ContentSerializer):
         source="thread_type",
         choices=[(val, val) for val in ["discussion", "question"]]
     )
+    abuse_flagged_count = serializers.SerializerMethodField(required=False)
```

Comment:
nit: flagged_abuse_count

Reply:
The existing field is called `abuse_flagged`, so I am trying to maintain compatibility with that.